### PR TITLE
Fix internal input updating after UI changes in HTML editor (T692735)

### PR DESCRIPTION
**Pull request by KsyushaM**

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

File: js/ui/html_editor/ui.html_editor.js [hunk 1]
```diff
@@ -345,19 +345,18 @@ const HtmlEditor = Editor.inherit({
     _optionChanged: function(args) {
         switch(args.name) {
             case "value":
-                !this._isEditorUpdating && this._setSubmitValue(args.value);
-
-                if(!this._quillInstance) {
+                if(this._quillInstance) {
+                    if(this._isEditorUpdating) {
+                        this._isEditorUpdating = undefined;
```
Comment (churkin) [inline js/ui/html_editor/ui.html_editor.js]: `this._isEditorUpdating = false;` ?

**Review by churkin — CHANGES_REQUESTED** [review]

File: js/ui/html_editor/ui.html_editor.js (hunk 1 above)
Comment (KsyushaM) [reply js/ui/html_editor/ui.html_editor.js]: Instead of "delete".
Is this matter?

Comment (churkin) [reply js/ui/html_editor/ui.html_editor.js]: As I see, you've boolean variable `isEditorUpdating` that can be `true` or `false` only. Editor is in the updating mode or not, I do not see third state there. I suggest to use `true` and `false` values for the boolean vars. Then I see `delete this.booleanVar` or `this.booleanVar = undefined` I begin looking for why was it really necessary, it is a bad practice.

Comment (KsyushaM) [reply js/ui/html_editor/ui.html_editor.js]: OK. I've already fixed it

**Review by mrjohn0011 — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Review by churkin — APPROVED** [review]

**Pull request merged**